packages/storybook-readme/src/services/transformEmojis.js

```diff
@@ -0,0 +1,19 @@
+const emoji = require('node-emoji');
+const EMOJI = /:\+1:|:-1:|:[\w-]+:/g;
+
+const transformEmojis = doc => {
+  const resultMap = {};
+  let result;
+  while ((result = EMOJI.exec(doc)) !== null) {
+    resultMap[result] = true;
+  }
+  Object.keys(resultMap).forEach(result => {
+    doc = doc.replace(
+      new RegExp(result, 'g'),
```

Comment:
Would be nice to move this regex up to the top of the file, so there's only 1 single regex instance and it's re-used.